test/node/conversation-labels.js

```diff
@@ -55,11 +49,12 @@ describe('Authentication', () => {
         assert(returnedLabel.value === addedLabelsHT[labelIdToEdit].value && returnedLabel.labelId === addedLabelsHT[labelIdToEdit].labelId);
     });
 
-    it('should assign a label to the first conversatrion', async () => {
+    it('should assign a label to the first conversation', async () => {
         let conversation = await client.getConversations({numberOfConversations: 1});
         conversation = conversation[0];
         const labelIdsToAssign = Object.keys(addedLabelsHT);
         const res = await client.assignLabels(conversation.convId, labelIdsToAssign);
+        await helper.expectEvents(client, ['conversationUserDataChanged']); 
```

Comment:
At least check that the conversationUserDataChanged is about the same conversation. Maybe even check that its the correct label. For that use a predicate. See example here https://github.com/circuit/circuit-sdk/blob/master/test/browser/directcall.js#L36